scrum/scrumguide

```diff
@@ -0,0 +1,167 @@
+ Negotiation = Development Team + Product Owner
+ Each Sprint =~ project no longer than 1 month long.
+ 
+Accomplish something
+ Has a goal of what to build
+ Has a design
+ Flexible plan
+ Result = Product Increment
+ 
+Only the Product Owner can cancel, before the time-box is over. May do under the influence of Stakeholders, Development Team or Product Owner.
+
+
+Sprint Planning:
+Maximum of 8 hours for a one-month Sprint. Shorter event for shorter sprints. For planning the work to be performed during the Sprint.
+
+It answers:
+
+What can be delivered?
+How will work be achieved?
+
+By the end of Sprint Planning, the Development Team should be able to explain how it intends to self-organise to accomplish the Goal and deliver the Increment.
+
+Sprint Backlog:
+Development Team self-organises to undertake (commit) the work in the Sprint Backlog
+
+During planning and
+Throughout the Sprint
+
+Daily Scrum:
+15 minutes time-box. Regardless team size or Sprint length.
+Internal meeting for the Development Team (others must not disturb) Every day.
+Same place, same time.
+Plan for the next 24 hours.
+Optimises collaboration and performance.
+Key Inspect and Adapt meeting.
+
+The structure is set by the Development Team
+
+Sprint Review:
+4 hours for one month Sprint. Shorter event for shorter sprints.
+End of Sprint (before Retrospective).
+What was done in the Sprint.
+Inspect the Product Increment.
+Adapt the Product Backlog.
+Scrum Team + Stakeholders.
+Collaborate on next things to be done to optimise value.
+
+Sprint Retrospective:
+3 hours for one month Sprint. Shorter event for shorter sprints.
+Formal opportunity for Inspection and Adaptation.
+Scrum Team inspect itself.
+Plan improvements to do next Sprint.
+Make next Sprint more effective and enjoyable.
+Plan how to increase product quality.
+Adapt the Definition of Done if
+Necessary.
+Appropriate.
+Not in conflict with organisational standards.
+
+Scrum Artefacts:
+Artefacts represent Work or Value.
+Opportunity to Inspect and Adapt.
+Designed to maximise transparency.
+
+Product Backlog:
+Ordered list of everything needed in the product.
+It is the single source of requirements.
+Belongs solely to the Product Owner.
+
+Sprint Backlog:
+Product Backlog items selected for the Sprint, plus a plan for delivering them = forecast by the Development Team about:
+What functionality will be in the next increment and
+Work needed to deliver it as a "Done" increment
+Makes visible the work necessary to deliver the Sprint Goal.
+It is a highly visible real-time picture of the Development Team work.
+It belongs solely to the Development Team.
+
+Increments:
+
+Sum of items completed during this Sprint	+ Value of increments from previous Sprints.
+
+Definition of Done :
+It guides the Development Team on knowing how many items it can select during Sprint Planning.
+As Scrum Team matures, Definition of Done evolves to higher quality and may uncover work to be done in previously "Done" increments.
```

Comment:
I added my comments to the  document .Modifications are recommended for better view